pallets/invulnerables/src/lib.rs

```diff
@@ -62,6 +72,17 @@ pub mod pallet {
         /// Maximum number of invulnerables.
         type MaxInvulnerables: Get<u32>;
 
+        /// A stable ID for a collator.
+        type CollatorId: Member + Parameter;
+
+        /// A conversion from account ID to collator ID.
+        ///
+        /// Its cost must be at most one storage read.
+        type CollatorIdOf: Convert<Self::AccountId, Option<Self::CollatorId>>;
```

Comment:
What's the purpose of this? It is set to "IdentityCollator", which does nothing because `CollatorId` is defined to be `AccountId`.

Reply:
we did the same that cumulus does, because yes it does nothing but it could be that collatorId is different to accountId. We can remove it if we are sure we will not change it..+